people_msgs/msg/Person.msg

```diff
@@ -4,4 +4,4 @@ geometry_msgs/Point velocity
 float64             reliability
 string[]            tagnames
 string[]            tags
-
+people_msgs/PositionMeasurement position_measurement
```

Comment:
The other changes make sense, but why this one?

Reply:
Sorry, the last commit has nothing to do with this PR. I needed the covariance from the `PositionMeasurement` so I included it in the tracked message. Is it possible to exclude that commit from the PR?